sphinx_gherkindoc/parsers/pytest_bdd.py

```diff
@@ -1,19 +1,21 @@
 """Helper functions for writing rST files."""
 from collections import namedtuple
+from typing import List, Optional, Union
 import pathlib
 
 import pytest_bdd.feature
 
 from .base import BaseModel
 
+BlankString = str
```

Comment:
This seems like it doesn't help much. mypy won't distinguish the difference, so if you use a non-blank string it won't work.

I think this reveals that we have places where we really should be using `None` to show that something isn't there and have return types of `Optional[<thing>]`. For the sake of not changing the code, I say just drop this extra layer and stick with `str`, then we can change to typings that make more sense as a FF if we care to.

Reply:
I agree that we're probably better off moving those to returning `None` long-term rather than a blank string. I did this simple alias for now as a way to make sure we remember the expectations, but if you feel strongly about dropping it, I can -- just felt like a good safety reminder until we do refactor. (I'd originally set this up as a `Literal[""]` type, but that's only supported in `Python3.8+`)

Reply:
Yeah I don't feel strongly. Fine enough with me.

Reply:
At the least, this puts us in a spot to cleanly find-and-fix the `BlankString` locations when we *do* decide to make that change. :)